.github/workflows/nightly-build.yml

```diff
@@ -52,10 +73,9 @@ jobs:
       - name: Download natives and build project
         uses: actions/download-artifact@v4
         with:
-          pattern: natives-*
```

Comment:
I removed the pattern key here, because we use all uploaded artifacts anyway. Since we also include the msi artifact, we would have to check how to extend the pattern here in the case, we want to stick with it.